Makefile

```diff
@@ -175,3 +181,15 @@ PHONY: .shell
 		-w $(PWD) \
 		--user $$(id -u):$$(id -g) \
 		$(BUILDIMAGE)
+
+## help: Display help information
+help: Makefile
```

Comment:
Question: Does this mean that the output will only be shown if the `Makefile` changes? Should `help` be a `.PHONY` target?